src/react-native/07-debugging-devtools/debugging-devtools.md

```diff
@@ -53,19 +48,27 @@ At this point, the DevTool is not connected. To connect the DevTool to the appli
 adb shell input keyevent 82
 ```
 
-This will refresh the connection. 
+This will refresh the connection.
+
+### Exercise 1
+
+- Explore Components:
+  - Use the `Components` tab to explore the hierarchical structure of the React Native components in the application.
+  - Expand and collapse component trees to understand the component composition.
+  - Hover over individual components to highlight them in the browser.
+- Inspect Props and State:
+  - Select a component from the component tree to inspect its props and state.
+  - View the current props and state values, and observe how they affect the component's rendering.
+- Modify Props:
+  - Experiment with modifying props directly from the DevTools panel.
+  - Change prop values and observe how they affect the component's appearance or behavior in real-time.
 
 ## Objective 2: React Native Debugger
 
 We will be setting up React Native Debugger to help us debug:
 
-**React Native Debugger**
-
 <img alt="Screenshot of the React Native JavaScript Inspector interface. It includes common DevTools tabs such as Console, Sources, Memory, and Profiler." src="../../static/img/react-native/07-debugging-devtools/react-native-debugger.png" style="max-width: 100%;"/>
 
-### Concept TODO
-TODO
-
```

Comment:
Same thing here: let’s explain what it is and what it enables us to do.

Also, does their UI allow for light mode? If so, then I’d take a screenshot of that and not use dark mode.